shopify/resources/order.py

```diff
@@ -19,3 +20,16 @@ def transactions(self):
 
     def capture(self, amount=""):
         return Transaction.create({"amount": amount, "kind": "capture", "order_id": self.id})
+
+    def add_fulfillment(self, fulfillment):
+        if not self.id:
+            self.save()
+        fulfillment.attributes['order_id'] = self.id
+
+        result = fulfillment.save()
+        if result:
+            if not isinstance(self.attributes.get('fulfillments'), Iterable):
+                self.attributes['fulfillments'] = []
```

Comment:
I'm not sure I follow. If the order already has a fulfillment, it will already be an Iterable and this line will be skipped.